psp_notes.md

```diff
@@ -0,0 +1,86 @@
+## Steps required befor PSP flow can be started:
```

Comment:
Can you update this file and ask for a review again please?

Reply:
I was planning to just delete it after properly updating the README.